### DISCUSSION TITLE
Translation in several languages

### DISCUSSION THREAD

**Discussion by TheSirC** [tosdr/tosdr.org#177] [Ideas] [answered]
Hello,
Thank you for your amazing work! I would like to contribute to the project by providing translation. Where can I do it?

**Comment by evilcel3ri** [comment]
Hey ! Thanks for your interest in tosdr ! Right now we are rebooting a lot of our tools and I'm not sure where to begin to with translations... I asked the core team and will come back to you asap :)

**Comment by hugoroy** [comment]
What do you want to translate? The software (i.e. to launch a tosdr version in another language, for a specific country for instance) or the content (i.e. translate submissions made by other people)?

**Comment by TheSirC** [comment]
Both? I am a French native but I speak English fluently so I wanted to put that to good use!

**Comment by evilcel3ri** [comment]
That's amazing ! Look we may have something right now you could help to, we will launch a campaign next week and we'll need translations. I would be amazing if you could help us with that. 

The thing about the software is that it is current under development and I'd feel bad to make you work on unfinished data.

What do you think ?

**Comment by TheSirC** [comment]
That sounds great !

**Comment by evilcel3ri** [comment]
Ok, get in touch with me, my email : piks3l@mailoo.org :)

**Comment by michel-thomas** [comment]
I'm french too and I can help translating the software and the content.

But I wonder if the content should be translated as is or if the software should manage geographical specificity for each service. In other words, I am not sure that a service X has the same agreements in USA (CloudAct conformity), in Europe (RGPD conformity) or in Russia...
I'm pretty sure that regional version of agreements differs, and law is applied based on the texts written in user native language first.
So the question is: will it be "only a translation" or a localization of the content?

**Comment by michielbdejong** [comment]
Good question! Do you know of an example of a big service whose terms are different depending on continent? The easy way to deal with that would be to make it two different services on a case-to-case basis first (e.g. "Facebook (Europe)", "Facebook (outside Europe)") and then when we have two or three of those, it becomes obvious we want to display that more sensibly on https://tosdr.org, and also make the browser extensions aware of it.

**Comment by michel-thomas** [comment]
I can't name a service that have different conditions according the localization, but I'm pretty sure it exists.

But event if it's the same conditions in all country, it is not the same language. Example with Facebook [France](https://www.facebook.com/legal/terms?locale=fr_FR) and [USA](https://www.facebook.com/legal/terms?locale=en_US). I guess the content is the same but the terms the user will accept are the one from its country in its language.

Your idea to localize by the name is good to apply quickly, but I think this could become quite messy if each country add its language (US english, UK english, french France, french Canada, spanish, italian, polish...).
I will start like you said by naming "Service (France)" waiting a better solution. Thanks for the idea.

**Comment by michielbdejong** [comment]
Found one! :) https://www.whatsapp.com/legal?eea=0 vs https://www.whatsapp.com/legal?eea=1 will try out how we can deal with that, if the differences are small then maybe we should keep it as one review of one service that happens to have several docs

**Comment by augusto-herrmann** [comment]
I am a native Portuguese speaker and would like to contribue translations, too. Website, content, software...

What you do in this project is amazing and I always tell other people about it.

**Comment by JustinBack** [comment, marked as answer]
Translations are now possible in Crisp!

https://github.com/JustinBack/CrispCMS-ToS-DR
I am open for help regarding translations!
